### PR TITLE
update secret names and doc for osa

### PR DESCRIPTION
**Pull request by mjudeikis**
ptal: @kargakis , @jim-minter 

cc: @openshift/sig-azure

### CODE REVIEW

File: projects/azure/README.md [hunk 1]
```diff
@@ -25,6 +25,15 @@ ci-operator --config ci-operator/config/openshift/openshift-azure/master.json --
 
 # Secret rotation
 
+We use 2 types of secrets. Both of them contains same data.
```
Comment (kargakis) [inline projects/azure/README.md]: s/contains/contain/

Comment (kargakis) [reply projects/azure/README.md]: > Both of them contains same data. 

They don't actually contain the same data. More precisely the format differs. We should probably call out that the env based secret is what we use for deploying clusters in Azure (so it's a drop-in config file for Openshift) whereas the file based approach is what ci-operator expects in order to source it in its environment.

Comment (kargakis) [reply projects/azure/README.md]: Sorry, azure-purge is using the env based secret. Is it the same as azure.conf? If not disregard my previous comment.

Comment (mjudeikis) [reply projects/azure/README.md]: One is just `export FOO=BAR` in the file, second one is `--from-literal=foo=${BAR}`. Same credentials, same variables just format.

**Comment by kargakis** [comment]
Unfortunately it's needed by the ci-operator. We could potentially have an
init container in the ci-operator pod that reads the azure-purge secret and
writes down the ci-operator format. We'd still need to keep the azure-purge
secret in two different place (azure and ci namespace) until the developer
productivity team creates the secret mirror controller.

**Comment by petr-muller** [comment]
About the rebase request:

We have mass-renamed all ci-operator configuration files from `.json` to `.yaml` in https://github.com/openshift/release/pull/1336, because YAML is what ci-operator consumes now. All the references of these files needed to be changed too. JSON is a YAML subset so that the actual file content can stay like it is (we'll eventually mass-reformat them to the block style).

**Comment by mjudeikis** [comment]
@petr-muller This was fast :) will fix this accordingly

**Comment by mjudeikis** [comment]
ping @jim-minter

**Comment by jim-minter** [comment]
thanks for the explanation @mjudeikis - go ahead as you were :)

/hold cancel

**Comment by mjudeikis** [comment]
Post-merge these need to be executed:
```
oc get secret cluster-secrets-azure -o yaml --export  -n azure | sed s/cluster-secrets-azure/cluster-secrets-azure-env/ | oc apply -f -
 oc get secret cluster-secrets-azure-temp -o yaml --export -n azure | sed s/cluster-secrets-azure-temp/cluster-secrets-azure-file/ | oc apply -f - -n ci
```

**Comment by mjudeikis** [comment]
ping @stevekuznetsov @kargakis

File: projects/azure/README.md [hunk 2]
```diff
@@ -25,6 +25,15 @@ ci-operator --config ci-operator/config/openshift/openshift-azure/master.yaml --
 
 # Secret rotation
 
+We use 2 types of secrets. Both of them contain same data.
```
[the remark this answers is not in the document]
Comment (mjudeikis) [reply projects/azure/README.md]: In can :) I will be happy to argue with anybody who is not you at any time when somebody comes and pick this as maintenance job :D

**Comment by mjudeikis** [comment]
lets merge it after lunch as if I left some typos in renaming all stuff will go down :D

**Pull request merged**